checkOverlaps.C

```diff
@@ -1,28 +1,47 @@
-void checkOverlaps(TString filename)
+void checkOverlaps(TString filename, Int_t method =0)
 {
-    TGeoManager *geo = TGeoManager::Import(filename);
-
-    cout<<"======================== Checking Geometry ============================="<<endl;
-    geo->CheckGeometry();
-    cout<<"========================       Done!       ============================="<<endl;
-
-    cout<<"======================== Checking Overlaps ============================="<<endl;
-    geo->CheckOverlaps(1e-5,"s");
-    geo->PrintOverlaps();
-    cout<<"========================       Done!       =============================\n\n\n"<<endl;
-
-    if(not gROOT->IsBatch()){
-        TObjArray* overlaps=geo->GetListOfOverlaps();
-        for(int i=0; i<overlaps->GetEntries(); i++){
-            TObject* overlap=overlaps->At(i);
-            cout<<"========================  Drawing Overlaps ============================="<<endl;
-            cout<<"================= Overlap messages will duplicate below ================"<<endl;
-            cout<<"=================     Overlaps are in units of cm       ================"<<endl;
-            TCanvas* c = new TCanvas();
-            overlap->Draw("");
-            TCanvas* cogl = new TCanvas();
-            overlap->Draw("ogl");
-            cout<<"========================       Done!       =============================\n\n\n"<<endl;
-        }
-    }
+ TGeoManager *geo = TGeoManager::Import(filename);
+ cout<<"======================== Checking Geometry ============================="<<endl;
+ //geo->CheckGeometry();
+ //cout<<"========================       Done!       ============================="<<endl;
+ if(method==1){
+   cout<<"======================== Checking Overlaps with samplig method ============================="<<endl;
+   cout<<"Enter Method: "<<endl;
+   geo->CheckOverlaps(1e-6,"s");
+   geo->PrintOverlaps();
+   cout<<"========================       Done!       =============================\n\n\n"<<endl;
+   if(not gROOT->IsBatch()){
+     TObjArray* overlaps=geo->GetListOfOverlaps();
+     for(int i=0; i<overlaps->GetEntries(); i++){
+       TObject* overlap=overlaps->At(i);
+       cout<<"========================  Drawing Overlaps ============================="<<endl;
+       cout<<"================= Overlap messages will duplicate below ================"<<endl;
+       cout<<"=================     Overlaps are in units of cm       ================"<<endl;
+       TCanvas* c = new TCanvas("c","sampling method");
+       overlap->Draw("");
+       TCanvas* cogl = new TCanvas();
+       overlap->Draw("ogl");
+       cout<<"========================       Done!       =============================\n\n\n"<<endl;
+     }
+   }
+}
```

Comment:
This curly bracket is at the wrong indentation. Also, your first indentation level is one space, while the second level is two spaces. Please make this consistent.